trtools/annotaTR/annotaTR.py

```diff
@@ -256,6 +256,44 @@ def GetLocusKey(record, match_on=RefMatchTypes.locid):
     else:
         raise ValueError("Invalid match_refpanel_on=%s"%match_on)
 
+def GetAlleleCount(record):
```

Comment:
```suggestion
def GetAlleleCount(record: cyvcf2.Variant):
```
How similar is this method to [`tr_harmonizer.GetAlleleCounts`](https://github.com/gymrek-lab/TRTools/blob/6bdc596db02dba1e38f7ab11fba8cb20377e0eaf/trtools/utils/tr_harmonizer.py#L1420)? Maybe you can also describe the differences between the two methods in the function signature of this one?